content/comm/conferences/index.en.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "R-Ladies Conference Participation"
+menuTitle: "Conference Participation"
+weight: 110
+---
+
+R-Ladies participates in conference sessions such as contributed talks,
+panel discussions, networking events, poster presentations, and booths to 
+increase awareness of our organization.
+
+## Organizing a conference event
+
+If you have an R-Ladies member interested in leading a conference
+session, please notify us at `info@rladies.org` for our records and so that
+we may offer additional support if needed.
+
+If you need assistance identifying a candidate R-Ladies member to lead a session,
+please email us with the template below. 
+
+It would be appreciated if fees associated with the conference could be waived
```

Comment:
Should this be re-inforced somehow by explicitly stating the organization and the invited speaker will not pay for the session?
And that further financial support might help the invited speaker?

Reply:
I second that - adding more details here will be useful!

Reply:
How about something like: 

>Speakers invited to represent R-Ladies should have all conference fees waived or reimbursed. If travel is involved, the conference should further support the speaker financially by reimbursing travel, lodging, and meals.